packages/node/src/ClientAuthentication.ts

```diff
@@ -59,6 +59,8 @@ export default class ClientAuthentication {
     // could return to a previous redirect URL that contains OIDC params that
```

Comment:
Not part of this PR, but there's a comment here mentioning the browser even though this is the Node package :)